README.md

```diff
@@ -42,9 +39,14 @@ for our CI purposes.
 ```
 Once the above variables are set and the /payload directory has been populated, run the following script:
 ```shell script
-hack/run-ci-e2e-test.sh
+hack/run-ci-e2e-test.sh -k "openshift-dev"
 ```
-
+Additional flags that can be passed to `hack/run-ci-e2e-test.sh` are
+- `-s` to skip the deletion of Windows nodes that are created as part of test suite run
+- `-n` to represent the number of Windows nodes to be created for test run
+- `-k` to represent the key pair that will be used during e2e run. This private key path in this pair should match with 
+       what we have in `KUBE_SSH_KEY_PATH` The default value points to `libra` which we use in our CI
```

Comment:
This is confusing. From the example above it seems you are asking for the key pair name in the cloud account but you are not being clear about the connection with `KUBE_SSH_KEY_PATH`.

Please add the example you have included in the commit message here too.

Reply:
`This private key path in this pair should match with what we have in `KUBE_SSH_KEY_PATH``. You mean I should elaborate on that?

Reply:
What does `This private key path in this pair` even mean? You should make it less confusing. Reading the explanation it seemed like `-k` can be used to specify the key itself. You need to call out that this specific to key-pair names in the AWS account and that it should map to the private key specified in `KUBE_SSH_KEY_PATH`.